config/routes.rb

```diff
@@ -18,10 +17,12 @@
   resources :volunteers do
     get :seeking_clients, on: :collection
     get :find_client, on: :member, to: 'assignments#find_client'
+    get :volunteer_hours, on: :member
     resources :journals
   end
   resources :volunteer_emails
   resources :profiles, except: [:destroy, :index]
+  resources :hours
```

Comment:
but what is different in this implementation, so that it shouldn't be done like that. We have records related to volunteer and assignments. basically.